micropub.php

```diff
@@ -330,7 +332,15 @@ private static function generate_args() {
    * Generates and returns a post_content string suitable for wp_insert_post()
    * and friends.
    */
-  private static function generate_post_content() {
+  public static function generate_post_content($args) {
+    // If the theme declares it supports microformats2, pass the content through
+    if (current_theme_supports('microformats2')) {
+      return $args;
+		}
+    // Disable if Kind Taxonomy is enabled
```

Comment:
I could move that code back to the Post Kinds plugin. I had it there originally.